src/create2/interfaces/ICreate2Factory.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.21;
 
-interface IPredictiveDeployer {
+interface ICreate2Factory {
```

Comment:
Consider reordering these to match the order of functions in the contract, similar to the ICreate3Factory interface.